lenskit-core/src/main/java/org/grouplens/lenskit/data/dao/RatingFile.java

```diff
@@ -1,42 +0,0 @@
-/*
- * LensKit, an open source recommender systems toolkit.
- * Copyright 2010-2014 LensKit Contributors.  See CONTRIBUTORS.md.
- * Work on LensKit has been funded by the National Science Foundation under
- * grants IIS 05-34939, 08-08692, 08-12148, and 10-17697.
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU Lesser General Public License as
- * published by the Free Software Foundation; either version 2.1 of the
- * License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
- * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
- * details.
- *
- * You should have received a copy of the GNU General Public License along with
- * this program; if not, write to the Free Software Foundation, Inc., 51
- * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
- */
-package org.grouplens.lenskit.data.dao;
-
-import org.grouplens.lenskit.core.Parameter;
-import org.grouplens.lenskit.data.text.EventFile;
-
-import javax.inject.Qualifier;
-import java.io.File;
-import java.lang.annotation.*;
-
-/**
- * Rating file for {@link SimpleFileRatingDAO}.
- *
- * @author <a href="http://www.grouplens.org">GroupLens Research</a>
- * @since 2.1
- */
-@Qualifier
-@Parameter(File.class)
-@Retention(RetentionPolicy.RUNTIME)
-@Target({ElementType.PARAMETER, ElementType.METHOD})
-@Documented
-public @interface RatingFile {
```

Comment:
Yes, probably an oversight. The only component that it configured was deprecated.